docs/Makefile

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Doxygen requires these packages:
+# doxygen (for doxygen)
+# texlive-full (for pdflatex and float.sty, used by PDF generation)
+
+TARGET=refman
+PDF_TARGET=$(TARGET)/latex/refman.pdf
```

Comment:
Do we need SRC (input directories for docxygen to generate documentation). Where should we add this.

Reply:
Add source directories to the `INPUT` line in file `docs/Doxyfile`.
File Doxyfile is not shown because it is large--click on it to view the comments.
I'll add a comment to the Makefile.